build

```diff
@@ -4,7 +4,7 @@ NAME="coreos-metadata"
 ORG_PATH="github.com/coreos"
 REPO_PATH="${ORG_PATH}/${NAME}"
 VERSION=$(git describe --dirty)
-GLDFLAGS="-X main.version \"${VERSION}\""
+GLDFLAGS="-X main.version=\"${VERSION}\""
```

Comment:
I assumed not based on
https://github.com/coreos/rkt/commit/2b1a60db047f5b2a408586e01678c461446a5c80

On Mon, Sep 28, 2015 at 5:42 PM, Vito Caputo <notifications@github.com>
wrote:

> In build
> <https://github.com/coreos/coreos-metadata/pull/6#discussion_r40623581>:
>
> > @@ -4,7 +4,7 @@ NAME="coreos-metadata"
> >  ORG_PATH="github.com/coreos"
> >  REPO_PATH="${ORG_PATH}/${NAME}"
> >  VERSION=$(git describe --dirty)
> > -GLDFLAGS="-X main.version \"${VERSION}\""
> > +GLDFLAGS="-X main.version=\"${VERSION}\""
>
> doesn't = work everywhere?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/coreos-metadata/pull/6/files#r40623581>.
>